docs/queryworkers/sql/advanced-commands.md

```diff
@@ -0,0 +1,125 @@
+---
+sidebar_position: 10
+title: Advanced SQL Queries
+---
+
+## Case Sensitivity
```

Comment:
Two things:
- Always leave an empty line under headings.
- Do not stack headings. In this case, the H1 generated by the title will be stacked atop this H2. Add a short intro explaining what this topic is about above Case Sensitivity.